src/status_im/subs/communities.cljs

```diff
@@ -243,7 +243,8 @@
                 emoji
                 can-view?
                 can-post?
-                token-gated?]}]]
+                token-gated?
+                hideIfPermissionsNotMet]}]]
```

Comment:
`:hide-if-permissions-not-met?`

Reply:
I'll look into the attribute's renaming flow again, But I am sure that's exactly what was destructured as.
Maybe the renaming didn't happen correctly.

Reply:
Fixed here https://github.com/status-im/status-mobile/pull/19378/commits/998b8c29ed15a11a9caa9ad3d61964a0ebb8e8ef